### ISSUE TITLE
JSON-LD: PUT API is non-functional

### ISSUE THREAD

**Issue by azaroth42** [archesproject/arches#4102]
**Describe the bug**

When attempting to create an instance of the simplest model with the PUT API, the response received is "500 '@type'"  with no body.

**To Reproduce**

Create a model with the CRM ontology.
Select the class to be E21 Person
Set the model to be active
Add a node, "label", type xsd:string, property rdfs:label, datatype string

Send the following API request

```
curl -v -H "Authorization: Bearer {your token}" -X PUT -d '{"@context":"http://linked.art/ns/v1/linked-art.json","id":"http://lod.example.org/museum/Person/0","type":"Person","label":"Test Person"}' http://localhost:8000/resources/
```

**Expected Behavior**

This should create a new Person instance using the model defined.

**Comment by apeters** [comment]
Currently with PUT you must supply tileid/nodeid values in order to update a resource.
Obviously, the HTTP 200 is incorrect and is a bug.

**Comment by azaroth42** [comment]
No, it's really non-functional if you don't supply the exact key "@type" containing the model UUID. It should at the very least respect the supplied @context.

You can test by creating a resource, setting the context in the UI to `https://linked.art/ns/v1/linked-art.json`, requesting the JSON-LD representation and then trying to post the exact same JSON back.

**Comment by apeters** [comment]
PUT should work if you first GET a resource.  Take that JSON-LD and update a value(s).
PUT that resource back and that should work.

**Comment by azaroth42** [comment]
Yup, and that doesn't work :( In this case because there isn't an `@type` key in the data due to the context definition.
